Content.IntegrationTests/Tests/Networking/ConnectTest.cs

```diff
@@ -55,10 +55,12 @@ public async Task TestConnect()
             var clEntityManager = client.ResolveDependency<IEntityManager>();
             var svEntityManager = server.ResolveDependency<IEntityManager>();
 
-            var lastSvEntity = svEntityManager.GetEntities().Last();
-            var lastClEntity = clEntityManager.GetEntity(lastSvEntity.Uid);
+            var lastSvEntity = svEntityManager.GetEntities().Last().Uid;
+            var lastClEntity = clEntityManager.GetEntity(lastSvEntity).Uid;
```

Comment:
isn't this kinda useless? as in cant you just use the serveruid on both, since we dont have a seperation rn. and wont this break with seperation anyways, so doing it like this in advance also doesnt make much sense

Reply:
idk man, I'm just fixing tests

Reply:
This isn't useless as it used to get the IEntity and check the transform there instead. That's bad and it didn't work.

Reply:
this really is a nitpick but i do wanna understand what you mean since the unresolve:
I get what it did before, but now the test is only using the uid, meaning you might as well just use the serveruid for both, since they are equal anyways. right?

Reply:
sure